__test__/server/api/campaign.test.js

```diff
@@ -778,6 +779,82 @@ describe("Reassignments", async () => {
     expect(texterCampaignDataResults2.data.assignment.allContactsCount).toEqual(
       29
     );
-    console.timeEnd('fullfunction');
-  }, 10000);  // long test can exceed default 5seconds
+    console.timeEnd("fullfunction");
+  }, 10000); // long test can exceed default 5seconds
+});
+
+describe("Bulk Send", async () => {
```

Comment:
can you add a test above here that confirms that when NOT_IN_USA and/or ALLOW_SEND_ALL are not enabled that bulkSend fails or errors?